### PR TITLE
[FR] Traduction de Loosely Coupled Architecture into French

### LINKED ISSUE #2029: [FR]  Localization of Loosely Coupled Architecture into french.
Localize Loosely Coupled Architecture in French

### PR DESCRIPTION
**Pull request by huats**
### Describe your changes

This PR localizes Loosely Coupled Architecture concept to French

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2029 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/loosely-coupled-architectures.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+status: Completed
+category: Property
+tags: ["fundamental", "architecture", "property"]
+---
+
+L'architecture faiblement couplée est un type d'architecture où les différents composants
+d'une application sont construits indépendamment les uns des autres (c'est le paradigme opposé
+des [architectures fortement couplées](/tightly-coupled-architectures/)).
+Chaque composant, parfois identifié comme un [microservice](/microservices/), est construit pour effectuer une tâche spécifique d'une manière qui lui permet d'être utilisé par d'autres services.
+Cette approche est souvent plus longue à mettre en oeuvre que l'architecture fortement couplée
```
Comment (sestegra) [inline content/fr/loosely-coupled-architectures.md:12]: s/oeuvre/œuvre/

File: content/fr/loosely-coupled-architectures.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+mais elle a plusieurs avantages, en particulier lorsque l'application change d'échelle.
```
Comment (sestegra) [inline content/fr/loosely-coupled-architectures.md:13]: Il est conseillée de mettre une virgule avec `mais`.

File: content/fr/loosely-coupled-architectures.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Les application faiblement couplées permettent aux équipes de développer des fonctionnalités, de les déployer et les mettre à l'échelle de manière indépendante, ce qui permet aux organisations d'itérer rapidement au sein des composants de manière individuelle.
```
Comment (sestegra) [inline content/fr/loosely-coupled-architectures.md:15]: s/application/applications/

Comment (sestegra) [inline content/fr/loosely-coupled-architectures.md:15]: Dans la traduction de `reliability`, j'ai traduit `individual component` par `composant unitaire`.
Qu'en penses-tu ?

**Review by sestegra — CHANGES_REQUESTED** [review]

**Review by sestegra — APPROVED** [review]

**Review by Krast76 — APPROVED** [review]

**Pull request merged**